h2o-py/h2o/model/models/word_embedding.py

```diff
@@ -36,7 +36,7 @@ def find_synonyms(self, word, count=20):
         >>> print(synonyms)
         """
         j = h2o.api("GET /3/Word2VecSynonyms", data={'model': self.model_id, 'word': word, 'count': count})
-        return OrderedDict(sorted(zip(j['synonyms'], j['scores']), key=lambda t: t[1], reverse=True))
+        return dict(zip(j['synonyms'], j['scores']))
```

Comment:
The result is sorted on the Java side; there is no need to sort it again.

Reply:
Are we 100% sure that the serialization and de-serialization guarantee the order on all cases? 

I would reconsider the sort.

The python dict does not guarantee the order for python 3.6 though. It is guaranteed for >= 3.7 and for 3.6 is stated as implementation detail. Since we are sure that its meant to me sorted. I would use OrderedDict anyway and make it future proof.

Reply:
Thanks, @valenad1, for suggesting to keep sorting on the client side. We can't guarantee the deserialization to keep the order so that I will keep it here.